kymatio/scattering3d/backend/torch_backend.py

```diff
@@ -0,0 +1,326 @@
+import torch
+import warnings
+
+BACKEND_NAME = 'torch'
```

Comment:
Is this variable used??

Reply:
Let's keep it for now and leave it later